### DISCUSSION TITLE
C# DateOnly & TimeOnly

### DISCUSSION THREAD

**Discussion by bauersystems** [ravendb/ravendb#13669] [General]
I am looking to use RavenDB in a new project, however the project needs to utilise DateOnly & TimeOnly.

Does anyone know whether RavenDB handles C#'s DateOnly & TimeOnly well?

**Comment by ppekrol** [comment]
We are working on a full support for those types. You can track the progress on our bugtracker here: https://issues.hibernatingrhinos.com/issue/RavenDB-17250. There is also a draft PR available here: https://github.com/ravendb/ravendb/pull/13667

**Reply by bauersystems** [reply]
Great, thanks. I'll keep an eye on it and see how it progresses.
